charts/internal/alicloud-infra/templates/main.tf

```diff
@@ -14,7 +14,13 @@ resource "alicloud_key_pair" "publickey" {
 resource "alicloud_vpc" "vpc" {
   name       = "{{ required "clusterName is required" .Values.clusterName }}-vpc"
   cidr_block = "{{ required "vpc.cidr is required" .Values.vpc.cidr }}"
+
+  timeouts {
```

Comment:
why don't have timeouts setting for `update`?

Reply:
It's not supported: https://www.terraform.io/docs/providers/alicloud/r/vpc.html#timeouts